app/mailers/subscription_mailer.rb

```diff
@@ -23,4 +23,15 @@ def notify_note_liked(node, user)
     @footer = feature('email-footer')
     mail(to: node.author.email, subject: subject).deliver
   end
+
+  def notify_tag_added(node, tag)
+    @node = node
+    given_tags = node.tags
+    users_to_email = tag.followers_who_dont_follow_tags(given_tags)
```

Comment:
Are we sure we don't include the author of the tag in this list? Best add a test to ensure you're not getting a notification for a tag you yourself created.